docs/decisions/0001-support-glob-pattern.rst

```diff
@@ -0,0 +1,54 @@
+Establishing a Repository for Translations Management
+#####################################################
+
+Description
+***********
+Glob pattern support has been added to ``atlas``.
+
+Example
+*******
+
+``atlas pull --expand-glob translations/*/done`` pulls
+``atlas pull translations/DoneXBlock/done`` if it exists.
+
+This is useful for instance to pull XBlocks in which the directory name
+isn't known but the module name is.
+
+This is an alternative to the `edx-platform-links`_ strategy because
+``git sparse-checkout --no-cone`` don't support glob patterns across links.
+
+
+Dismissed alternatives
+**********************
+
+1. Support links again:
+-----------------------
+
+Previous tests on unknown git version showed to support links in April 2023,
+but it had no automated tests and broke somewhere during the
+`git v2.25.1 version support`_ we've added.
+
+The sparse-checkout part is already complex, so we'll not touch it.
+
+2. Make complete copies of the translations in `edx-platform-links`_
+---------------------------------------------------------------------
+This is the most viable and simple alternative to glob patterns.
+
+If it turns out the ``--expand-glob`` option is flaky to work with, we'll
+fallback to this.
+
+This _may_ complicate the GitHub Transifex App sync process which we prefer
+to keep simple and avoid adding additional step to ensure the
+`edx-platform-links`_ have full copies of the XBlock and plugins translations
+instead of symbolic links.
+
+3. Use the `edx-platform-links`_ to store the original files
+------------------------------------------------------------
+
+This is a viable option as well but needs to refactor the Transifex
+configuration which could reset the translations.
+
+Therefore, we'll only fallback to this if the glob patterns has problems.
```

Comment:
It's not fully clear to me how these alternatives would accomplish what is being done with `--expand-glob`

It seems each of the alternatives has a great description of *why* we decided against it, but not a full explanation of *how* it would work if we decided to go that route.

I also think the lines about which alternative we'll fall back to if the glob expansion is problematic can be removed, we can reference the alternatives we rejected and decide on a new path forward if that time comes.

Reply:
Good points. Both addressed.